3.processing_features/scripts/feature_select_sc.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform feature selection on normalized merged single cells for each plate
+
+# ## Import libraries
+
+# In[1]:
+
+
+import sys
+import pathlib
+import os
+
+import pandas as pd
+from pycytominer import feature_select
+from pycytominer.cyto_utils import output
+
+sys.path.append("../utils")
+import extraction_utils as sc_utils
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+# output directory for feature selected data
+output_dir = pathlib.Path("./data/feature_selected_data")
+# if directory if doesn't exist, will not raise error if it already exists
+os.makedirs(output_dir, exist_ok=True)
+
+# dictionary with each run for the cell type
+plate_info_dictionary = {
+    "Plate_1": {
+        # path to parquet file from normalize function
+        "normalized_path": str(pathlib.Path("./data/normalized_data/Plate_1_sc_norm.parquet"))
+    },
+    "Plate_2": {
+        # path to parquet file from normalize function
+        "normalized_path": str(pathlib.Path("./data/normalized_data/Plate_2_sc_norm.parquet"))
+    },
+    "Plate_3": {
+        # path to parquet file from normalize function
+        "normalized_path": str(pathlib.Path("./data/normalized_data/Plate_3_sc_norm.parquet"))
+    },
+    "Plate_3_prime": {
+        # path to parquet file from normalize function
+        "normalized_path": str(pathlib.Path("./data/normalized_data/Plate_3_prime_sc_norm.parquet"))
+    }
+}
+
+# ## Perform feature selection
+# 
+# The operations that we are using for feature selection are:
+# 
+# - `variance_threshold`: creates a list of exlcuded features that have very low varience of values between single cells
+#   
+# - `correlation_threshold`: creates a list of excluded features with a correlation to at least one other feature greater than the default threshold (`threshold=0.9`)
+#   
+# - `blocklist`: creates a list of excluded features using the [standard blocklist file](https://github.com/cytomining/pycytominer/blob/master/pycytominer/data/blocklist_features.txt) for CellProfiler features from Pycytominer
+# 
+# For more information regarding these operations, please visit [the Pycytominer operations folder](https://github.com/cytomining/pycytominer/tree/master/pycytominer/operations) on GitHub.
+# To view how `blocklist` works, please visit [the seperate file](https://github.com/cytomining/pycytominer/blob/a5ae6c81a275b692ef5d4c85cfeb37696bf69242/pycytominer/cyto_utils/features.py#L13) for that function.
```

Comment:
Consider removing this block if it is no longer needed.

Reply:
This is the documentation for a user to know why these operations were chosen (recommended by two different reviewers). It is easier to document in the notebook over putting this somewhere else.